webpack.config.js

```diff
@@ -0,0 +1,22 @@
+var webpack = require('webpack')
+
+module.exports = {
+  entry: './index.js',
+  output: {
+    path: __dirname,
+    filename: 'fuzzyfind.min.js'
+  },
+  plugins: [
+    new webpack.optimize.UglifyJsPlugin({
+      compress: { warnings: false },
+    }),
+    new webpack.BannerPlugin([
```

Comment:
Never mind. I found out what it does. But is this common? Do people really leave big comments on top of minified JS?